### PR TITLE
Add Shisen CI

### LINKED ISSUE #3: Add CI Using GitHub Actions
Add a continuous integration for this project using [GitHub Actions](https://github.com/features/actions), see [this guide](https://docs.github.com/en/actions/guides/building-and-testing-nodejs) for more information on how to setup CI for a Node.js project using GitHub Actions.

### PR DESCRIPTION
**Pull request by BryanYehuda**
Closes #3

### CODE REVIEW

**Comment by threeal** [comment]
Test failed, see https://github.com/ichiro-its/shisen-rvue/actions/runs/626711512.

File: .github/workflows/shisen_ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+    with:
+      node-version: '14'
```
Comment (threeal) [inline .github/workflows/shisen_ci.yml]: I think the error is in this line, the `with:` is not aligned with the `uses:`.

File: .github/workflows/shisen_ci.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - run: npm install
+    - run: npm test
```
Comment (threeal) [inline .github/workflows/shisen_ci.yml]: Also, don't forget to add an empty line before the end of the file.

**Review by threeal — CHANGES_REQUESTED** [review]
See individual comment for the requested changes.

File: .github/workflows/shisen_ci.yml (hunk 1 above)
Comment (BryanYehuda) [reply .github/workflows/shisen_ci.yml]: Will be fixed shortly, thank you for the suggestions

File: .github/workflows/shisen_ci.yml (hunk 2 above)
Comment (BryanYehuda) [reply .github/workflows/shisen_ci.yml]: Will be fixed shortly, thank you for the suggestions

File: .github/workflows/shisen_ci.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+      node-version: '14'
```
Comment (threeal) [inline .github/workflows/shisen_ci.yml]: Indent the `node-version` too, as it lies inside the `with:` parameter, see [this guide](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith).

**Review by threeal — CHANGES_REQUESTED** [review]
Workflow fail, see in [here](https://github.com/ichiro-its/shisen-rvue/actions/runs/626802555). Check the comment for the suggested change.

File: .github/workflows/shisen_ci.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+    - run: npm install
+    - run: npm test
```
Comment (threeal) [inline .github/workflows/shisen_ci.yml:23]: Run the `npm run build` as specified in the [`package.json`](https://github.com/ichiro-its/shisen-rvue/blob/master/package.json). Run it after the `npm install` and before the `npm test`.

**Review by threeal — CHANGES_REQUESTED** [review]
The action succeeds, but the build process hasn't been specified. See the comment for the suggested change.

**Review by threeal — APPROVED** [review]
LGTM

**Pull request merged**